pkg/template/apis/template/validation/validation.go

```diff
@@ -71,15 +70,6 @@ func ValidateTemplateInstance(templateInstance *templateapi.TemplateInstance) (a
 			allErrs = append(allErrs, field.Required(field.NewPath("spec.secret.name"), ""))
 		}
 	}
-	if templateInstance.Spec.Requester == nil {
-		allErrs = append(allErrs, field.Required(field.NewPath("spec.requester"), ""))
-	} else if templateInstance.Spec.Requester.Username == "" {
-		allErrs = append(allErrs, field.Required(field.NewPath("spec.requester.username"), ""))
```

Comment:
why do we no longer require this?  Don't we still need to know who we're instantiating the template on the behalf of?